h2o-docs/src/product/data-science/aggregator.rst

```diff
@@ -4,11 +4,17 @@ Aggregagtor
 Introduction
 ~~~~~~~~~~~~
 
-The H2O Aggregator method is a clustering-based method for reducing a numerical/categorical dataset into a dataset with fewer rows. Aggregator maintains outliers as outliers, but lumps together dense clusters into exemplars with an attached count column showing the member points.
+The H2O Aggregator method is a clustering-based method for reducing a numerical/categorical dataset into a dataset with fewer rows. If the dataset has categorical columns, then for each categorical column, Aggregator will:
+
+ 1. Accumulate the category frequencies.
+ 2. For the top 1,000 or fewer categories (by frequency), generate dummy variables (called one-hot encoding by ML people, called dummy coding by statisticians).
```

Comment:
I was looking at Aggregator code https://github.com/h2oai/h2o-3/blob/master/h2o-algos/src/main/java/hex/aggregator/Aggregator.java and can't find references to 1000. Probably missing something obvious. Is this a constant or we can somehow tune this number of categories (1000) ?

Reply:
It’s hard wired. The idea is that after 1,000 dummy variables, the eigenvectors wouldn't change much by adding more, plus you could run out of memory and/or time.

Reply:
Makes sense. Thank you for clarifying